crates/re_data_store/src/log_db.rs

```diff
@@ -203,11 +203,16 @@ impl LogDb {
         self.entity_db.tree.num_timeless_messages()
     }
 
+    pub fn len(&self) -> usize {
```

Comment:
`len` is such an over-loaded word. Let's be explicit

```suggestion
    pub fn num_rows(&self) -> usize {
```